### DISCUSSION TITLE
how to use command line '-c'

### DISCUSSION THREAD

**Discussion by 1s1u** [mpromonet/webrtc-streamer#592] [Q&A]
please teach how to use commadn line '-c' .
 In my opinion, the sslketcert is composed of key and cert, so i dont know how to  use a param instead of tow params.

Best Regards,
harry.

**Comment by mpromonet** [comment]
Hi @1s1u ,

The file should contains both, the key & cert.

Best Regards,
Michel.

**Reply by 1s1u** [reply]
Could you give an  example plz? Just Concatenate the key and cert with symbols “&”？ @mpromonet 

Best regard，
Harry

**Reply by mpromonet** [reply]
just concatenate both files in a standard way, see https://github.com/civetweb/civetweb/blob/master/docs/OpenSSL.md
